client/views/room/MessageList/components/UrlPreview/OEmbedHtmlPreview.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react';
+
+import OEmbedCollapseable from './OEmbedCollapseable';
+import type { PreviewMetadata } from './PreviewList';
+
+const OEmbedImagePreview = ({ html, ...props }: PreviewMetadata): ReactElement => (
+	<OEmbedCollapseable {...props}>{html && <div dangerouslySetInnerHTML={{ __html: html }} />}</OEmbedCollapseable>
```

Comment:
the html is already sanitized? if not, can we sanitize with something like dompurify?

Reply:
I believe we have a sort of filter/whitelist on the backend.